wallet/src/plan/planner.rs

```diff
@@ -333,7 +333,7 @@ impl<R: RngCore + CryptoRng> Planner<R> {
         // For any remaining provided balance, make a single change note for each
         let self_address = fvk
             .incoming()
-            .payment_address(source.unwrap_or(AddressIndex::Numeric(0)))
+            .payment_address(source.unwrap_or(AddressIndex::default()))
```

Comment:
Here the source looks like it was formerly an AddressIndex being used as a logical account, probably the method should be changed so that source is a u32 account index, rather than being an option? There was a note about that change on the original issue and I think this might be running up against that change.